packages/uikit/package.json

```diff
@@ -83,50 +108,57 @@
     "rollup": "^2.70.1",
     "rollup-plugin-node-builtins": "^2.1.2",
     "storybook": "^7.0.7",
-    "styled-components": "^5.3.3",
+    "styled-components": "^6.0.7",
     "themeprovider-storybook": "^1.7.2",
     "ts-jest": "^27.1.3",
     "viem": "^1.2.9",
     "vite": "^4.3.1",
+    "vite-plugin-dts": "^3.5.3",
+    "vite-tsconfig-paths": "^4.0.3",
     "vitest": "^0.27.2",
     "wagmi": "^1.3.10"
   },
   "peerDependencies": {
-    "@pancakeswap/farms": "workspace:*",
-    "@pancakeswap/hooks": "workspace:*",
-    "@pancakeswap/localization": "workspace:*",
-    "@pancakeswap/pools": "workspace:*",
-    "@pancakeswap/sdk": "workspace:*",
-    "@pancakeswap/swap-sdk-core": "workspace:*",
-    "@pancakeswap/token-lists": "workspace:*",
-    "@pancakeswap/tokens": "workspace:*",
-    "@pancakeswap/ui": "workspace:*",
-    "@pancakeswap/utils": "workspace:*",
-    "@pancakeswap/v3-sdk": "workspace:*",
     "@sentry/nextjs": "^7.0.0",
     "js-cookie": "*",
     "next": "*",
     "next-seo": "*",
+    "next-themes": "^0.2.1",
     "react": "^18.2.0",
     "react-device-detect": "*",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-transition-group": "*",
     "remark-gfm": "*",
-    "styled-components": "^5.3.3",
+    "styled-components": "^6.0.7",
     "viem": "^1.2.9"
   },
   "dependencies": {
-    "@pancakeswap/ui": "workspace:*",
+    "@pancakeswap/farms": "workspace:*",
+    "@pancakeswap/hooks": "workspace:*",
+    "@pancakeswap/localization": "workspace:*",
+    "@pancakeswap/pools": "workspace:*",
+    "@pancakeswap/sdk": "workspace:*",
+    "@pancakeswap/swap-sdk-core": "workspace:*",
+    "@pancakeswap/token-lists": "workspace:*",
+    "@pancakeswap/tokens": "workspace:*",
+    "@pancakeswap/utils": "workspace:*",
```

Comment:
let's move @pancakeswap/utils to devDependencies as it's not published. And probably need to bundle it into the uikiit? For other pkgs we add it into `noExternal`

Reply:
seems like vite detect it as no external
I was on purpose keeping utils private to enable inline bundling